Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -62,6 +66,49 @@ theorem tendstoInMeasure_iff_norm [SeminormedAddCommGroup E] {l : Filter ι} {f
   simp_rw [TendstoInMeasure, dist_eq_norm]
 #align measure_theory.tendsto_in_measure_iff_norm MeasureTheory.tendstoInMeasure_iff_norm
 
+-- The ae-limit is ae-unique.
+theorem tendsto_ae_unique {α ι E : Type*} [TopologicalSpace E] [T2Space E]
+    {x : MeasurableSpace α} {μ : Measure α} {g h : α → E} {f : ι → α → E} {l : Filter ι} [l.NeBot]
+    (hg : ∀ᵐ ω ∂μ, Filter.Tendsto (fun i => f i ω) l (𝓝 (g ω)))
+    (hh : ∀ᵐ ω ∂μ, Filter.Tendsto (fun i => f i ω) l (𝓝 (h ω))) : g =ᵐ[μ] h := by
+  filter_upwards [hg, hh] with ω hg1 hh1 using tendsto_nhds_unique hg1 hh1
+
+/-- This notion is helpful for finite measures since we don't have to deal with the
+possibility that some set measures to ∞ -/
+def TendstoInMeasure' [Dist E] {_ : MeasurableSpace α} (μ : Measure α) (f : ι → α → E)
```

Comment:
I'm not convinced that this definition is useful. Can you explain a bit more why it is needed?

Reply:
It boils down to having a coercion `NNRead.toReal` but not a coercion `ENNReal.toReal`. Since the former exists, it is possible to compute stuff more easily when using `TendstoInMeasure'` rather than `TendstoInMeasure`. Of course there are other ways to avoid having this definition, but ir really helped me computing things faster.

Reply:
I think rather than making a new definition, it is better to replace the right hand side of `TendstoInMeasure_of_FiniteMeasure` by ` ∀ ε, 0 < ε → Tendsto (ENNReal.toNNReal ∘ (fun i => (μ { x | ε ≤ dist (f i x) (g x) }))) l (𝓝 0)`

Reply:
This is also a good idea. I not deleted the definition and followed the suggestion.